command/output.go

```diff
@@ -0,0 +1,29 @@
+package command
+
+import (
+	"fmt"
+)
+
+type Output struct {
+	Text       string
+	ExitStatus int
+}
+
+func NewErrorOutput(err error) *Output {
```

Comment:
why the lack of symmetry? Write receives the outputWriter and writes to it. But this one returns the Output, and lets the caller to do the write